docs/api.md

```diff
@@ -670,6 +670,25 @@ Adds a function which would be invoked in one of the following scenarios:
 
 The function is invoked after the document was created but before any of its scripts were run. This is useful to amend JavaScript environment, e.g. to seed `Math.random`.
 
+An example of using preloading some functions from an external file would be following,
+
+```js
+// preload.js
+module.exports = function(){
+    console.log("Adding some fake languages to navigator property");
+    
+    // overwrite the `languages` property to use a custom getter
+    Object.defineProperty(navigator, "languages", {
+      get: function() {
+        return ["en-US", "en", "c", "js"];
+      }
+    });
+}
+
+// on the puppeteer script, assuming the preload.js file is in same folder of our script
+await page.evaluateOnNewDocument(require("./preload"));
```

Comment:
I'd drop the `require` part: it adds unnecessary indirection.
It also assumes commonJS modules, which is opinionated.

Other then that, great example!

Reply:
Is there any other easy way to preload an external file?

Reply:
@entrptaher sure, you can use `fs` for this: `fs.readFileSync('filepath.js', 'utf8')`. Eventually the es6 imports would work as well.

Reply:
oh, that, I was wondering if you meant not to use any external file.
Let me fix these styles and things when I get time.

Reply:
@entrptaher I suggested not to bother with an external file since it has nothing to do with the API itself.